src/lib/settings/initialSettings.tsx

```diff
@@ -263,6 +264,10 @@ export function createSettings() {
           )
         },
       }),
+      namedViews: new Setting<NamedView[]>({
+        defaultValue: [],
+        validate: (v) => true,
```

Comment:
I think this needs the "only used in projects" or whatever flag?

Reply:
`hideOnLevel: 'app',`

Reply:
oh okay, I didn't realize we generate 1 settings then flag it for project or app level I thought we would be creating a settings for both not proxying 1 object for both states. Let me add this flag and test locally.

Reply:
```
/** The levels available to set settings at.
 * `project` settings are specific and saved in the project directory.
 * `user` settings are global and saved in the app config directory.
 */
export type SettingsLevel = 'user' | 'project'
```

I think it would be `hideOnLevel: 'user'` `app` is not a valid string.

Reply:
updated.

Reply:
ah yeah makes sense